mock/mockimpl/kvproc/operations.go

```diff
@@ -948,8 +948,14 @@ func (e *Engine) MultiMutate(opts MultiMutateOptions) (*MultiMutateResult, error
 			return nil, ErrLocked
 		}
 
+		nextSeqNo, err := e.db.NextSeqNo(doc.VbID)
```

Comment:
I'm not entirely sure of the ramifications of doing this. For a period we would have no doc/change for the seqno but then later could.